crates/sui-framework/packages/deepbook/sources/clob.move

```diff
@@ -480,16 +482,13 @@ module deepbook::clob {
                     balance::join(&mut pool.quote_asset_trading_fees, quote_balance_filled);
                     balance::join(&mut base_balance_filled, locked_base_balance);
 
-                    event::emit(OrderFilled<BaseAsset, QuoteAsset> {
-                        pool_id: *object::uid_as_inner(&pool.id),
-                        order_id: maker_order.order_id,
-                        is_bid: false,
-                        owner: maker_order.owner,
-                        total_quantity: maker_order.quantity,
-                        base_asset_quantity_filled: filled_base_quantity,
-                        base_asset_quantity_remaining: maker_base_quantity,
-                        price: maker_order.price
-                    })
+                    emit_order_filled<BaseAsset, QuoteAsset>(
+                        *object::uid_as_inner(&pool.id),
+                        maker_order,
+                        filled_base_quantity,
+                        filled_quote_quantity - filled_quote_quantity_without_commission,
```

Comment:
Should we defend against underflow here, or add a comment explaining why this can never happen?

Reply:
@sblackshear These two places won't cause underflow given how `filled_quote_quantity` and `filled_quote_quantity_without_commission` are calculated. We'll add comments on this.